lib/password_reset_token/password_reset_token_service.ex

```diff
@@ -0,0 +1,93 @@
+defmodule Pairmotron.PasswordResetTokenService do
```

Comment:
So I put this in it's own lib folder to sort of prepare for the 1.3 directory structure. This file would essentially be the context for the `Pairmotron.PasswordResetToken` schema.